mixins/hotjar.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  computed: {
+    $hotjarScript() {
+      if (this.$config.hotjar && this.$config.hotjar.id && this.$config.hotjar.sv) {
+        return { src: `https://static.hotjar.com/c/hotjar-${this.$config.hotjar.id}.js?sv=${this.$config.hotjar.sv}`, async: true };
```

Comment:
Use the complete tracking code and consider refactoring this in a plugin rather than a mixin

Reply:
Agreed, and done, on both counts.